### PR TITLE
chore: Promote unstable to rc

### PR DESCRIPTION
**Pull request by deregtd**
Promote unstable to rc, created by `dt release promote`

**DO NOT MERGE**

### CODE REVIEW

File: templates/internal/appName/k8s/activity.go.tpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // {{ stencil.ApplyTemplate "copyright" }}
-{{- $_ := file.SetPath (printf "internal/%s/kubernetes.go" .Config.Name) }}
+  {{- $_ := file.SetPath (printf "internal/%s/kubernetes.go" .Config.Name) }}
```
Comment (pbr0ck3r) [inline templates/internal/appName/k8s/activity.go.tpl:2]: Nit, extra space here?

Comment (deregtd) [reply templates/internal/appName/k8s/activity.go.tpl:2]: huh, that's weird.  the {{- right after it will nullify it, but prolly should fix that in master

Comment (deregtd) [reply templates/internal/appName/k8s/activity.go.tpl:2]: https://github.com/getoutreach/stencil-golang/pull/204

**Review by pbr0ck3r — APPROVED** [review]
Looks good

**Pull request merged**